ExporterTest/Makefile

```
@@ -0,0 +1,23 @@
+CFLAGS = -Wall -O2 -g -std=c++17
```

Since this is a C++ thing, this should be CXXFLAGS, right?
```suggestion
CXXFLAGS = -Wall -O2 -g -std=c++17
```